package.json

```diff
@@ -23,11 +29,10 @@
     "eslint-config-standard": "^17.0.0",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-unicorn": "^43.0.0",
+    "husky": "^4.3.8",
```

Comment:
@GenericP3rson I see that you've added `husky` but it's not configured. Are you planning to add more commits or can we remove it?

Reply:
We can remove it! I installed Husky to find any additional ESLint errors.